tests/functional/traits/traits_with_missing_region.json

```diff
@@ -0,0 +1,95 @@
+{
+  "generated_by": {
+    "program": "augur",
+    "version": "18.1.0"
```

Comment:
It may be best to remove the version here, as this could potentially cause Augur to complain in the future with version mismatch.
```suggestion
```

Reply:
Good eye! We've resolved this issue in functional tests by [excluding the `version` field in our diffs for JSON files](https://github.com/nextstrain/augur/blob/7f968ce7a0ebf1226bf1a5e99717f0175ae55159/scripts/diff_jsons.py#L16). This happens implicitly/by default, since it is such a common pattern.

Reply:
I didn't mean that it would cause problems in tests but that it causes augur to error with "input version incompatible".

I realize now that this here is an _output_ file for which this is not a problem. My concern relates to _input_ files and there ain't one here.